packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -88,11 +88,20 @@ export class InsecureUrlResolver implements IUrlResolver {
         }
     }
 
-    private resolveHelper(documentId: string) {
+    private resolveHelper(documentId: string, uri: string) {
```

Comment:
>uri: string [](start = 46, length = 11)

This is not the complete uri. But rather the componentPath or you could just use documentRelativePath from above.